docs/index.md

```diff
@@ -48,3 +48,32 @@ The actual sending of messages is managed via
 When schedules are uploaded, once-off `PeriodicTask` objects are created
 for each row in the schedule. These will call `tasks.send_msgs` with the
 appropriate arguments for the chat.
+
+
+### Creating schedules
+
+Click the "Download schedules" icon and save "schedules.xlsx". Then open
+it in your spreadsheet application.
+
+You will see that the sheet has three columns, "Days", "Message" and
+"Comment".
```

Comment:
Maybe also mention that there's one sheet per group?

Reply:
Somewhere we need to stress the order: the group must be created, then we must download the schedules to edit/modify before we upload to set the new schedule. Also need to include that the admin interface is used to set the state date and the time messages are sent out.